src/main/scala/io/iohk/ethereum/jsonrpc/CheckpointingService.scala

```diff
@@ -21,7 +22,7 @@ class CheckpointingService(
     lazy val bestBlockNum = blockchain.getBestBlockNumber()
     lazy val blockToReturnNum = bestBlockNum - bestBlockNum % req.checkpointingInterval
 
-    Future {
+    Task.fromFuture(Future {
```

Comment:
Why not `Task { ... }` then?

Reply:
This construction was removed everywhere, including PersonalService that was very very tricky.

Currently, Task.fromFuture is used:
* QAService.mineBlocks to get rid of that call we would need port Future => Task in `Consensus`; that could happen as separate PR (this PR solve more than half of the usages of Futures 334 vs 167)
* in a wrapper around ask pattern in AkkaTaskOps